helm-chart/templates/deployment.yaml

```diff
@@ -68,20 +93,56 @@ spec:
               value: {{ .Values.environment.db_user}}
             - name: "DB_ENV_MYSQL_PASSWORD"
               value: {{ .Values.environment.db_pwd}}
+            - name: "DB_PREFIX"
+              value: {{ .Values.environment.db_prefix}}
           ports:
             - name: http
               containerPort: {{ .Values.container.port }}
               protocol: TCP
+          {{- if .Values.startupProbe.enabled }}
+          startupProbe:
+            httpGet:
+              path: /webservices/status.php
+              port: http
+          {{- unset .Values.startupProbe "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if .Values.livenessProbe.enabled }}
           livenessProbe:
             httpGet:
               path: /webservices/status.php
               port: http
+          {{- unset .Values.livenessProbe "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if .Values.readinessProbe.enabled }}
           readinessProbe:
             httpGet:
               path: /webservices/status.php
               port: http
+          {{- unset .Values.readinessProbe "enabled" | toYaml | nindent 12 }}
+          {{- end }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+      {{- if and .Values.pvc.enabled .Values.mariadb.enabled }}
+      initContainers:
+        - name: init-config
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - 'bash'
+            - '-c'
+            - |
+              cd /var/www/html
+              if [ ! -e "confPV/production/config-itop.php" ];then
+                mkdir -p confPV/production
+                cp conf/production/config-itop.php confPV/production/
+                chown 33 confPV/production/config-itop.php
+              fi
+              exit 0
+          volumeMounts:
+            - mountPath: "/var/www/html/confPV"
+              name: {{ ternary "itop-pv" "itop-conf-pv" .Values.pvc.useOnePV }}
+              subPath: {{ ternary "conf" "" .Values.pvc.useOnePV }}
+      {{- end }}
```

Comment:
What is this section doing and what is it used for?

Reply:
I don't see why this config-itop.php needs to be copied somewhere?

Reply:
if we use PV, before install folder /var/www/htmlconf/production/ would be empty.
And we need to declare database connection params. So if we use [config-itop.php](https://github.com/Super-Visions/docker-itop/blob/master/3.1/config-itop.php), we can avoid this.
this make for convenient.

Reply:
Okay, what if this init container is already placing the credentials in the `config-itop.php` so, then the active iTop container also doesn't need those being set in the credentials?

Reply:
yes, if file exists - init container does nothing.

Reply:
Can you join the community on [Slack](https://join.slack.com/t/itopbycombodo/shared_invite/zt-uoat5wuf-C0g_1tqaeI9~EaqHxV7Q1Q)? Then we can maybe easier discuss some details..